SafeNeighbourhood/Pages/Index.cshtml

```diff
@@ -2,36 +2,57 @@
 @model IndexModel
 @{
     ViewData["Title"] = "Home page";
-    var welcomes = (QuickTypeVProperty.VacantProperty[]) ViewData["property"];
- 
+    var vacantProperties = (IEnumerable<QuicktypeProperty.PropertyData>)ViewData["VacantProperties"];
+    var neighborhoods = (List<QuicktypeNeighborhood.NeighborhoodData>)ViewData["Neighborhoods"];
 }
 
-<div class="text-center">
-    <h1 class="display-4">Welcome</h1>
-    <p>Learn about <a href="https://docs.microsoft.com/aspnet/core">building Web apps with ASP.NET Core</a>.</p>
+<div class="table-responsive">
+    <h2>Vacant Homes in Cincinnati by Neighborhood Cleanliness</h2>
+    <p>(Neighborhoods with the least number of complaints at the top.  Data since 2012)</p>
   
-    <table>
-        <tr>
-            <th>Neighbourhood</th>
-            <th>Longitude</th>
-            <th>Latitude</th>
-            <th>Street_Name</th>
-            <th>Street_Number</th>
-            <th>Property ID</th>
-        </tr>
-        @{
-            @foreach (QuickTypeVProperty.VacantProperty property in welcomes)
-            {
-                <tr>
-                    <td>@property.Neighborhood</td>
-                    <td>@property.Longitude</td>
-                    <td>@property.Latitude </td>
-                    <td>@property.StreetName</td>
-                    <td>@property.StreetNo</td>
-                    <td>@property.Uniqueid</td>
-                </tr>
-            }
-        }
+    <table class="table table-bordered">
+        @{foreach (QuicktypeNeighborhood.NeighborhoodData neighborhood in neighborhoods)
```

Comment:
Added a loop for each neighborhood, then, within each neighborhood, using a LINQ query to find all the vacant homes for that neighborhood, matching on the vacant home's neighborhood name (neighborhood.Neighborhood).